### ISSUE TITLE
Process substitution does not work because of "isfile" check

### ISSUE THREAD

**Issue by tomaszn** [garabik/grc#20]
Hi, it may be useful to use grc without a configuration file, e.g. in oneliners.

However, piping through 

```
... | grcat <(echo regexp=.*2015.*; echo colours=red)
```

fails, because "isfile" check returns False in such a case.

The fix is simple, just replace line:

```
if os.path.isfile(i+conffile_arg):
```

with:

```
if os.path.exists(i+conffile_arg) and not os.path.isdir(i+conffile_arg):
```

**Comment by garabik** [comment]
Yes, that's a fair point, I'll add the fix in the new version.

**Comment by garabik** [comment]
fixed in `devel` branch with 82ca58087cae842282651f15984f9c1134560d01
